include/morphio/iterators.h

```diff
@@ -52,31 +29,5 @@ class graph_iterator
     graph_iterator operator++(int);
 };
 
-// Declare the specializations
-template <>
-breadth_iterator::Iterator(const Section&);
-template <>
-upstream_iterator::Iterator(const Section&);
-template <>
-breadth_iterator::Iterator(const Morphology&);
-template <>
-upstream_iterator::Iterator(const Morphology&);
-template <>
-Section depth_iterator::operator*() const;
-template <>
-Section breadth_iterator::operator*() const;
-template <>
-Section upstream_iterator::operator*() const;
-template <>
-depth_iterator& depth_iterator::operator++();
-template <>
-breadth_iterator& breadth_iterator::operator++();
-template <>
-upstream_iterator& upstream_iterator::operator++();
-
-extern template class Iterator<std::stack<Section>>;
```

Comment:
no longer need #include <queue>

Confused as to why CI doesn't pick up such things, I just happened to check.